scripts/runOnDocker.sh

```diff
@@ -15,7 +15,7 @@ emulator -avd uiComparison -no-snapshot -gpu swiftshader_indirect -no-window -no
 PID=$(echo $!)
 
 ## server
-docker run --name=uiComparison nextcloudci/server 1>/dev/null &
+docker run --name=uiComparison nextcloudci/server:server-17 1>/dev/null &
```

Comment:
any particular reason for using -17? Is https://github.com/nextcloud/docker-ci/releases/tag/server-21 incompatible?

Reply:
That should not be part of this PR, I will remove it: https://github.com/nextcloud/android-library/pull/650